### PR TITLE
Added 526: Invalid SSL Certificate

### PR DESCRIPTION
**Pull request by itschasa**
passport != ssl_certificate

### CODE REVIEW

File: lib/statuses.js [hunk 1]
```diff
@@ -70,6 +70,7 @@ const statuses = {
   522: { code: 522, message: 'Connection Timed Out' },
   523: { code: 523, message: 'Origin Is Unreachable' },
   525: { code: 525, message: 'SSL Handshake Failed' },
+  526: { code: 525, message: 'Invalid SSL Certificate' },
```
Comment (rogeriopvl) [inline lib/statuses.js:73]:
```suggestion
  526: { code: 526, message: 'Invalid SSL Certificate' },
```

Comment (itschasa) [reply lib/statuses.js:73]: omg thats embarrasing

**Review by lucasmezencio — APPROVED** [review]

**Comment by itschasa** [comment]
my gitlens is saying that this needs a follow up

**Comment by lucasmezencio** [comment]
Hey @rogeriopvl and @itschasa, this extension is deprecated. Could you please do this for the "original" [HTTP Status Code extension](https://www.raycast.com/Alex_/http-status-codes)? :)

**Pull request closed without merging**